### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ptrdvrk**
self contained -> standalone extension
added the Troubleshooting section
added "Click on an edge" in the summary of features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +121,7 @@ Here are some tips:
 
 ### Share AppMap files for viewing in VSCode
 
-Generated AppMap files can be viewed by others with the AppMap extension. So, one option for sharing is to simply send the `appmap.json` file to your colleague.
+Generated AppMap files can be viewed by others with the AppMap extension. So, one option for sharing is to simply send the `appmap.json` file to your colleague. Another option is use of a git repository for sharing the `appmap.json` files with the members of your team.
```
Comment (kgilpin) [inline README.md:124]: This doesn’t seem like a good idea. We don’t really want people putting these big files in Git. How about Drive, Slack or Dropbox instead?

**Review by kgilpin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ptrdvrk) [reply README.md:124]: Ok, let me revert this change, the file sharing obviously needs a little more thinking and work.

**Review by kgilpin — APPROVED** [review]

**Comment by dustinbyrne** [comment]
In the future please run `npm install` (just the one time is fine) before committing. It'll enable a git post-commit hook that validates your commit message as a [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/). This is necessary for automated release. Otherwise it'll wait for a subsequent releasable change to push it out.

**Comment by ptrdvrk** [comment]
Will do Dustin. I am assuming `npm install` is required prior to the final commit before the PR is created?

**Comment by kgilpin** [comment]
Before any commit is made.

**Pull request merged**